app/modules/sightings/models.py

```diff
@@ -376,6 +376,39 @@ def delete_from_edm_by_guid(cls, current_app, guid, request):
             request_headers=request.headers,
         )
 
+    def get_jobs_json(self):
+        job_data = []
+        for job in self.jobs:
+            from app.modules.sightings.schemas import DetailedSightingJobSchema
+
+            schema = DetailedSightingJobSchema()
+            this_job = schema.dump(self.jobs[job]).data
+            this_job['job_id'] = job
+            job_data.append(this_job)
+
+        return job_data
+
+    def get_debug_sighting_json(self):
+        response = current_app.edm.get_dict('sighting.data_complete', self.guid)
+        if not isinstance(response, dict):  # some non-200 thing, incl 404
+            return response
+        if not response.get('success', False):
+            return response
+
+        from app.modules.sightings.schemas import DebugSightingSchema
+
+        schema = DebugSightingSchema()
+        edm_response = response['result']
+        for encounter in edm_response.get('encounters') or []:
+            # EDM returns strings for decimalLatitude and decimalLongitude
+            if encounter.get('decimalLongitude'):
+                encounter['decimalLongitude'] = float(encounter['decimalLongitude'])
```

Comment:
i am not 100% sure that edm _always_ returns a string for these -- but i have seen it.  however, i just tested, and `float(foo)` has no problem if `foo` is _already_ a float.  so not going to worry about the off chance edm returns floats!

Reply:
The change in this PR was just moving this logic into a separate function so that it could be reused.